### PR TITLE
Reduce data loaded before loading tests

### PR DESCRIPTION
**Pull request by timifasubaa**
This PR pushes down the CI build times for python tests by reducing the tables loaded before running the tests. 

@john-bodley @mistercrunch @michellethomas

### CODE REVIEW

**Comment by mistercrunch** [comment]
Ballpark before/after times?

**Comment by timifasubaa** [comment]
@mistercrunch  Shaved off about ~ 1 minute from the Python tests. From 5-6 mutes to 4 minutes

**Comment by michellethomas** [comment]
Not sure how others feel about removing the loading of these data sets completely. I wonder if we can have a flag for loading all data vs loading basic data sets. That way if some tests want to use all data they can.

**Comment by timifasubaa** [comment]
@michellethomas Agree. I updated the PR to only remove it completely from the tests loading. 
The `superset load_examples` command will continue to load all the data but when the `-t` flag or equivalent is added, it will add only the subset that's used in the tests.

**Comment by timifasubaa** [comment]
PING

File: superset/cli.py [hunk 1]
```diff
@@ -154,42 +154,43 @@ def load_examples_run(load_test_data):
...
 
-    print('Loading [Misc Charts] dashboard')
-    data.load_misc_dashboard()
+        print('Loading [Misc Charts] dashboard')
+        data.load_misc_dashboard()
 
-    if load_test_data:
-        print('Loading [Unicode test data]')
-        data.load_unicode_test_data()
+        if load_test_data:
```
Comment (michellethomas) [inline superset/cli.py]: This is nested under `if not load_test_data:`

Comment (timifasubaa) [reply superset/cli.py]: Done

**Pull request merged**